qutip/solver/heom/bofin_solvers.py

```diff
@@ -566,6 +568,17 @@ class HEOMSolver(Solver):
         If multiple baths are given, they must all be either fermionic
         or bosonic baths.
 
+    odd_parity : Bool
+        For fermionic baths only.  Default is "False".  If set to "True",
+        the RHS construction differs slightly (it implies the RHS is acting on
+        an initial density operator which has odd parity in terms of its
+        representation in fermionic annhilation operators). In other words, a
+        state with a well defined number of excitations has even parity, but
+        one with superpositions of the number of excitations has odd parity.
+        This is normally prohibited in 'normal' electronic systems, where
+        charge is conserved, but is useful for the construction of
+        electron spectrum (under linear response assumptions).
+
```

Comment:
Simon asked us to help distill and clarify this a little bit; how about:

" For fermionic baths only.  Default is "False".  "Parity" refers to the parity of the initial system state used with the HEOM. An odd parity state is one that can be made from an applying an odd number of fermionic creation operators to the vacuum density operator.  Physical systems have even parity, but allowing the generalization to odd-parity states allows one to calculate useful physical quantities like the system power spectrum or density of states. The form of the HEOM differs depending on the parity of the initial system state, so if this option is set to "True", a different RHS is constructed, which can then be used with a system state of odd parity. "